app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -217,8 +217,8 @@ void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex,
     // Note that Quest's runtime already shows the hamburger menu button when left
     // hand is facing head.
```

Comment:
Maybe update this comment, mentioning both hands?

Reply:
Done